Objective-C/CBLBlob.mm

```diff
@@ -275,7 +280,7 @@ - (BOOL) installInDatabase: (CBLDatabase*)db error:(NSError**)outError {
         uint8_t buffer[kReadBufferSize];
         NSInteger bytesRead = 0;
         _length = 0;
-        NSInputStream *contentStream = [self contentStream];
+        NSInputStream *contentStream = _initialContentStream;
```

Comment:
I think this is not correct. For example, if blob was initiated with content (Data). Probably need to get from _initialContentStream or [self contentStream] if _initialContentStream is null.

Reply:
Can we also check if we have unit tests to cover all constructors?

Reply:
it is working fine with the updated unit test. Since if it is initialized with content(Data), it will initiate via `success = c4blob_create(store, data2slice(_content), nullptr, &key, &err);`

Reply:
That is right.